test/FeatureSetSet_test.py

```diff
@@ -1,68 +1,95 @@
 """Basic FeatureSet tests."""
+from test.common_test import check_save_set_output
+from typing import Any
+
 import pytest
-from installed_clients.baseclient import ServerError
+from SetAPI.featureset.FeatureSetSetInterfaceV1 import FeatureSetSetInterfaceV1
+from SetAPI.generic.constants import INC_ITEM_INFO, INC_ITEM_REF_PATHS
 from SetAPI.SetAPIImpl import SetAPI
 from SetAPI.util import info_to_ref
-from SetAPI.generic.constants import INC_ITEM_INFO, INC_ITEM_REF_PATHS
 
+API_CLASS = FeatureSetSetInterfaceV1
 
-def test_save_feature_set_set(
-    featureset_refs: list,
+
+def save_featureset_set(
```

Comment:
I kitted this class out with tests like those in AssemblySet_basic_test.py. I would have done the other classes that allow empty sets in the same way, but didn't want this PR to get too big.

Reply:
THANK YOU